### PR TITLE
Static pages

### PR DESCRIPTION
**Pull request by Drewbrad4**
Small changes. Just added a little text and a link to the confirmation pages and changed the Not Found page.

### CODE REVIEW

File: app/javascript/components/Pages/Borrow.js [hunk 1]
```diff
@@ -4,44 +4,45 @@ import { Link } from 'react-router-dom';
...
-                        <h4>Author: {book.author}</h4>
-                        <p>Description: {book.description}</p>
-                        <p>Pages: {book.pages}</p>
-                        <p>Series: {book.series}</p>
-                        <p>Pick Up Date: {book.pick_up_date}</p>
-                        <p>Due Date: {book.due_date}</p>
+        {
+            books.map((book, index) => {
+                return (
+                    <div key={`book-${index}`} className="bookCard">
```
Comment (talk2MeGooseman) [inline app/javascript/components/Pages/Borrow.js:12]: You can just keep the key as `index` value here

**Review by talk2MeGooseman — APPROVED** [review]
Looks good, just a small comment.

File: app/javascript/components/Pages/Borrow.js (hunk 1 above)
Comment (sturdynut) [reply app/javascript/components/Pages/Borrow.js:12]: lol I think I told him to make it unique :)

Comment (talk2MeGooseman) [reply app/javascript/components/Pages/Borrow.js:12]: ha

**Pull request merged**